src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -741,7 +737,7 @@ class IrsPlan extends React.Component<
   // Jurisdiction Hierarchy Control
   /** onTableBreadCrumbClick - handler for drilldown table breadcrumb clicks to reset the table hierarchy */
   private onTableBreadCrumbClick = (e: MouseEvent) => {
-    preventDefault(e);
+    e.preventDefault();
```

Comment:
This is pretty low hanging fruit, if this is too much hassle then let's just not have util functions for these.

Reply:
`stopPropagation` and `stopPropagationAndPreventDefault` have to be click handler functions for the syntax to work:
```
```

Reply:
In the second example, I think we either replace `e.preventDefault()` with `preventDefault(e)` or we can just leave it as is.

Reply:
ok.